qcodes_contrib_drivers/drivers/StanfordResearchSystems/DG645.py

```diff
@@ -0,0 +1,283 @@
+import logging
+log = logging.getLogger(__name__)
+
+from qcodes import VisaInstrument
+import qcodes.utils.validators as vals
+
+class DG645(VisaInstrument):
+    """Qcodes driver for SRS DG645 digital delay generator.
+    """
+    CHANNEL_MAPPING = {
+        'T0': 0, 'T1': 1, 'A': 2, 'B': 3, 'C': 4,
+        'D': 5, 'E': 6, 'F': 7, 'G': 8, 'H': 9
+    }
+    OUTPUT_MAPPING = {'T0': 0, 'AB': 1, 'CD': 2, 'EF': 3, 'GH': 4}
+    PRESCALE_MAPPING = {'trig': 0, 'AB': 1, 'CD': 2, 'EF': 3, 'GH': 4}
+    TRIGGER_MAPPING = {
+        'internal': 0,
+        'ext_rising': 1,
+        'ext_falling': 2,
+        'single_ext_rising': 3,
+        'single_ext_falling': 4,
+        'single': 5,
+        'line': 6,
+    }
+    POLARITY_MAPPING = {'-': 0, '+': 1}
+    DISPLAY_MAPPING = {
+        'trig_rate': 0,
+        'trig_thresh': 1,
+        'trig_single_shot': 2,
+        'trig_line': 3,
+        'advanced_trig_enable': 4,
+        'trig_holdoff': 5,
+        'prescale_config': 6,
+        'burst_mode': 7,
+        'burst_delay': 8,
+        'burst_count': 9,
+        'burst_period': 10,
+        'channel_delay': 11,
+        'channel_output_levels': 12,
+        'channel_output_polarity': 13,
+        'burst_T0_config': 14
+    }
+    def __init__(self, name, address, **kwargs):
+        super().__init__(name, address, terminator='\r\n', timeout=10, **kwargs)
+        self.add_parameter('trig_holdoff',
+               label='Trigger holdoff',
+               unit='s',
+               get_cmd='HOLD?',
+               get_parser=float,
+               set_cmd='HOLD {}'
+        )
+
+        # Prescale parameters
+        for k, v in self.PRESCALE_MAPPING.items():
+            if v > 0:
+                self.add_parameter('phase_{}'.format(k),
+                       label='Prescale phase factor {}'.format(k),
+                       unit='',
```

Comment:
Let us get rid off empty units. :) I believe this should be done also in other PRs as well?